zio-sbt-ci/src/main/scala/zio/sbt/V.scala

```diff
@@ -5,7 +5,7 @@ object V {
     Map(
       "peter-evans/create-pull-request" -> "v6",
       "zio/generate-github-app-token"   -> "v1.0.0",
-      "pierotofy/set-swap-space"        -> "master",
+      "pierotofy/set-swap-space"        -> "49819abfb41bd9b44fb781159c033dba90353a7c", // 1.0,
```

Comment:
maybe it is better to mention the version here: "v1.0"

Reply:
I was not sure about the correct tagging. The github tag is 1.0 and the marketplace says v1.0. I just wanted to prevent the action from changing. I can try and make it v1.0 or 1.0.